swiftpkg/internal/swiftpkg_build_files.bzl

```diff
@@ -51,6 +51,16 @@ def _swift_target_build_file(pkg_ctx, target):
         "srcs": pkginfo_targets.srcs(target),
         "visibility": ["//visibility:public"],
     }
+
+    # Add macros as plugins
+    macro_target_labels = [target.label.repository_name + "//:" + target.label.name for target in pkg_ctx.pkg_info.targets if target.type == "macro"]
```

Comment:
Could we wrap this to stay [under 80 chars](https://bazel.build/rules/bzl-style#line-length)?

Reply:
037aa33